### PR TITLE
removing react-use package

### PR DESCRIPTION
**Pull request by Arpitha-Rajeev**

### CODE REVIEW

File: src/components/subscribemodal/index.tsx [hunk 1]
```diff
@@ -1,21 +1,20 @@
 import * as React from "react";
-import * as ReactUse from "react-use";
```
Comment (subhranshudas) [inline src/components/subscribemodal/index.tsx:2]: did we remove this from the package.json itself?? that was the main thing right?

File: src/hooks/useClickAway.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { RefObject, useEffect, useRef } from 'react';
+
+const defaultEvents = ['mousedown', 'touchstart'];
+
+export function on<T extends Window | Document | HTMLElement | EventTarget>(
```
Comment (subhranshudas) [inline src/hooks/useClickAway.ts]: remove exports here

File: src/hooks/useClickAway.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  obj: T | null,
+  ...args: Parameters<T['addEventListener']> | [string, Function | null, ...any]
+): void {
+  if (obj && obj.addEventListener) {
+    obj.addEventListener(...(args as Parameters<HTMLElement['addEventListener']>));
+  }
+}
+
+export function off<T extends Window | Document | HTMLElement | EventTarget>(
```
Comment (subhranshudas) [inline src/hooks/useClickAway.ts]: remove exports here

File: src/components/subscribemodal/index.tsx (hunk 1 above)
Comment (Arpitha-gr) [reply src/components/subscribemodal/index.tsx:2]: No we have not removed it from package.json.

Comment (Arpitha-Rajeev) [reply src/components/subscribemodal/index.tsx:2]: No we haven't removed it from package.json.

File: package.json [hunk 4]
```diff
@@ -17,8 +17,7 @@
   "dependencies": {
     "axios": "^0.27.2",
     "date-fns": "^2.28.0",
-    "html-react-parser": "^1.4.12",
-    "react-use": "^17.4.0"
+    "html-react-parser": "^1.4.12"
```
Comment (subhranshudas) [inline package.json:20]: anytime we change `package.json` we should have an updated `package-lock.json` **OR** `yarn.lock` if we are using `yarn`

**Pull request merged**